tests/openbook-twap.ts

```diff
@@ -126,7 +126,7 @@ describe("openbook-twap", () => {
       null,
       twapMarket,
       { confFilter: 0.1, maxStalenessSlots: 100 },
-      marketKP
+      marketKP.publicKey
```

Comment:
yeah, this is an issue because the latest npm version doesn't have [this](https://github.com/openbook-dex/openbook-v2/commit/f402b72a740d3fb1a9ca7832c15a08fc7a8b2d3b) change, so you'll need to do the yarn link thing